src/components/Attachments/AttachmentsPageHeader.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeft } from '@fortawesome/free-solid-svg-icons';
+import * as AppActions from 'components/App/AppActions';
+import * as AppConstants from 'components/App/AppConstants';
+import * as Actions from './AttachmentsActions';
+import * as Selectors from './AttachmentsSelectors';
+
+const StyledPageHeaderContainer = styled.div`
+  background-color: #1a1a1a;
+`;
+
+const StyledBackButton = styled.button`
+  width: 30px;
+  height: 30px;
+  border: none;
+  outline: none;
+  background: none;
+`;
+
+const AttachmentsPageHeader = () => {
+  const dispatch = useDispatch();
+  const canClearList = useSelector(Selectors.canClearUploadList);
```

Comment:
The page is named "attachments" but selector "canClear *upload* list".
I think would make more sense to call it UploadsPage then.
But you have used Attachments word in so many places... A lot of things to rename T_T.